### PR TITLE
Removed useless back to top links

### LINKED ISSUE #15431: Remove useless back to top links
There are some pages that has Back to top links, see [current search results])https://github.com/dotnet/docs/search?q=Back+to+top&type=Code).

I believe they existed in an old system that didn't have the current side bar that makes it easy to navigate in the article. They're not needed now.

### PR DESCRIPTION
**Pull request by srhsrhsrhsrh**
## Summary

Removed all instances of "[Back to top](#top)" from project 

Fixes #15431

### CODE REVIEW

File: docs/framework/interop/interop-marshaling.md [hunk 1]
```diff
@@ -42,8 +42,6 @@ ms.author: "ronpet"
   
  At the lowest level, both mechanisms use the same interop marshaling service; however, certain data types are supported exclusively by COM interop or platform invoke. For details, see [Default Marshaling Behavior](default-marshaling-behavior.md).  
   
- [Back to top](#top)
```
Comment (pkulikov) [inline docs/framework/interop/interop-marshaling.md]: With that PR, all `<a name="top"></a>` at the beginning of the files become useless.

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @sarahliii for creating this PR to remove those legacy back to top links. @pkulikov is right that we should remove the anchor from the top as well. 

I'd even take a step further and remove the menu on the top and the additional anchors from those articles since that's already generated in the new system. But I'd leave it up to you whether you'd like to address this on this PR or not.

**Comment by sarahliii** [comment]
> Thanks @sarahliii for creating this PR to remove those legacy back to top links. @pkulikov is right that we should remove the anchor from the top as well.
> 
> I'd even take a step further and remove the menu on the top and the additional anchors from those articles since that's already generated in the new system. But I'd leave it up to you whether you'd like to address this on this PR or not.

I'll work on removing the anchor for "top". I think I'll hold off on the further changes since I'm not so familiar with the system. (Wouldn't want to break anything!)

**Comment by mairaw** [comment]
Yeah including the intro sentence. Here’s an example:
722C226F-A559-4EC3-92AB-6F8BA8E502FD

But it’s totally fine. Let’s keep this one focused on the task at hand.

Don’t be afraid though! You shouldn’t be able to break anything and we can always revert PRs as well. Thank you!

**Comment by mairaw** [comment]
And I wouldn’t remove the anchors wholesale. We can clean some that seem useless and see if we get any build warnings.

File: docs/framework/misc/security-transparent-code-level-2.md [hunk 2]
```diff
@@ -99,8 +99,6 @@ The security attributes that are applied to a type also apply to the members tha
 
 - `SecuritySafeCritical`: The type or member is safe-critical. However, the type or member can be called from transparent (partially trusted) code and is as capable as any other critical code. The code must be audited for security.
```
Comment (mairaw) [inline docs/framework/misc/security-transparent-code-level-2.md]: Remove the top anchor from this file too please

File: docs/framework/misc/security-transparent-code.md [hunk 3]
```diff
@@ -43,8 +43,6 @@ Transparency was introduced in the .NET Framework version 2.0 to simplify the se
 > [!NOTE]
 > When you develop a partially trusted application, you have to be aware of the permission requirements for your target hosts. You can develop an application that uses resources that are not allowed by some hosts. This application will compile without error, but will fail when it is loaded into the hosted environment. If you have developed your application using Visual Studio, you can enable debugging in partial trust or in a restricted permission set from the development environment. For more information, see [How to: Debug a ClickOnce Application with Restricted Permissions](/visualstudio/deployment/how-to-debug-a-clickonce-application-with-restricted-permissions). The Calculate Permissions feature provided for ClickOnce applications is also available for any partially trusted application.
```
Comment (mairaw) [inline docs/framework/misc/security-transparent-code.md]: Same on this file

File: docs/framework/performance/interop-etw-events.md [hunk 4]
```diff
@@ -9,13 +9,19 @@ author: "mairaw"
 ms.author: "mairaw"
 ---
 # Interop ETW Events
-<a name="top"></a> Interop events capture information about Microsoft intermediate language (MSIL) stub generation and caching.  
+Interop events capture information about Microsoft intermediate language (MSIL) stub generation and caching.  
   
  This category consists of the following events:  
   
-- [ILStubGenerated Event](#ilstubgenerated_event)  
+- [Interop ETW Events](#interop-etw-events)
```
Comment (mairaw) [inline docs/framework/performance/interop-etw-events.md]: I’d roll back this change

File: docs/framework/performance/method-etw-events.md [hunk 5]
```diff
@@ -62,8 +62,6 @@ The following table shows the event data:
 |MethodFlags|win:UInt32|0x1: Dynamic method.<br /><br /> 0x2: Generic method.<br /><br /> 0x4: JIT-compiled code method (otherwise NGEN native image code).<br /><br /> 0x8: Helper method.|
 |ClrInstanceID|win:UInt16|Unique ID for the instance of CLR or CoreCLR.|
 
-[Back to top](#top)
-
 <a name="clr_method_marker_events"></a>
 
 ## CLR Method Marker Events
```
Comment (mairaw) [inline docs/framework/performance/method-etw-events.md]: Still has the anchor tag as well

File: docs/framework/performance/security-etw-events.md [hunk 6]
```diff
@@ -9,13 +9,19 @@ author: "mairaw"
 ms.author: "mairaw"
 ---
 # Security ETW Events
-<a name="top"></a> Security events are raised during strong name verification and Authenticode verification.  
+Security events are raised during strong name verification and Authenticode verification.  
   
  This category consists of the following events:  
   
-- [StrongNameVerificationStart_V1 and StrongNameVerificationStop_V1 Events](#strongnameverificationstart_v1_and_strongnameverificationstop_v1_events)  
+- [Security ETW Events](#security-etw-events)
```
Comment (mairaw) [inline docs/framework/performance/security-etw-events.md]: Please roll back this change

File: docs/standard/security/cryptographic-services.md [hunk 7]
```diff
@@ -30,15 +30,25 @@ ms.author: "mairaw"
...
 
 In the .NET Framework, the classes in the <xref:System.Security.Cryptography?displayProperty=nameWithType> namespace manage many details of cryptography for you. Some are wrappers for the unmanaged Microsoft Cryptography API (CryptoAPI), while others are purely managed implementations. You do not need to be an expert in cryptography to use these classes. When you create a new instance of one of the encryption algorithm classes, keys are autogenerated for ease of use, and default properties are as safe and secure as possible.
 
 This overview provides a synopsis of the encryption methods and practices supported by the .NET Framework, including the ClickOnce manifests, Suite B, and Cryptography Next Generation (CNG) support introduced in the .NET Framework 3.5.
 
 This overview contains the following sections:
 
-- [Cryptographic Primitives](#primitives)
+- [Cryptographic Services](#cryptographic-services)
```
Comment (mairaw) [inline docs/standard/security/cryptographic-services.md]: Please undo this change

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @sarahliii. There are still a few things to address before we can merge this.

**Comment by nemrism** [comment]
I'm surprised this subject is not on @NextTurn bucket list ;)
Nice 1st contribution @sarahliii

**Comment by mairaw** [comment]
I have resolved some merge conflicts that showed up during the day since they were complicated to resolve. Make sure to do git pull before you push any changes. Also, please let us know if you need any help to finish this PR. Thanks!

**Comment by NextTurn** [comment]
@nemrism I've got a bunch of other things to work on recently 😅.

**Comment by nemrism** [comment]
> @nemrism I've got a bunch of other things to work on recently 😅.

I'm also still trying to design a better solution to work on the #10653 subject!

**Review by mairaw — APPROVED** [review]
I've addressed my final feedback so we can merge this. Thanks @sarahliii!

**Pull request merged**